examples/tutorials/starting/analysis_1.py

```diff
@@ -396,8 +396,19 @@
 # 
 
 analysis.config.flux_points.source = "crab"
+# Example showing how to change the FluxPointsEstimator parameters:
+analysis.config.flux_points.energy.nbins = 5
+config_dict = {"selection_optional": "all",
+              "n_sigma": 2, #Number of sigma to use for asymmetric error computation
+              "n_sigma_ul": 3, #Number of sigma to use for upper limit computation
+              }
+analysis.config.flux_points.parameters = config_dict
+
 analysis.get_flux_points()
 
+# Example showing how to change the threshold on the signal significance (points vs upper limits)
+fp = analysis.flux_points.data
+fp.sqrt_ts_threshold_ul = 10
```

Comment:
Same here it is `sqrt(TS)` so you apply again 10 sigma here...
```suggestion
fp.sqrt_ts_threshold_ul = 3
```

Reply:
the same here